projects/1.16/assets/infernal-expansion/infernalexp/lang/zh_cn.json

```
@@ -1 +1,345 @@
+  "infernalexp.config.tooltip.piglinAttackVoline": "猪灵将攻击熔岩吞噬兽",
+  "infernalexp.config.tooltip.bruteAttackSkeleton": "猪灵蛮兵将攻击骷髅",
+  "infernalexp.config.tooltip.bruteAttackVoline": "猪灵蛮兵将攻击熔岩吞噬兽",
+  "infernalexp.config.tooltip.ghastAttackEmbody": "恶魂将射击嗜骨沙鬼",
+  "infernalexp.config.tooltip.ghastAttackVoline": "恶魂将射击熔岩吞噬兽",
+  "infernalexp.config.tooltip.ghastAttackSkeleton": "恶魂将射击骷髅",
+  "infernalexp.config.tooltip.ghastAttackGlowsquito": "恶魂将射击荧光蚊",
+  "infernalexp.config.tooltip.glowsquitoAttackDwarf": "荧光蚊和黑石侏儒将互相攻击",
+  "infernalexp.config.tooltip.glowsquitoAttackLuminous": "荧光蚊将攻击有荧光效果的实体",
+  "infernalexp.config.tooltip.dwarfAttackPiglin": "黑石侏儒将攻击猪灵",
+  "infernalexp.config.tooltip.dwarfAttackZombiePiglin": "黑石侏儒将攻击僵尸猪灵",
+  "infernalexp.config.tooltip.dwarfAttackSkeletalPiglin": "黑石侏儒将攻击骷髅猪灵",
+  "infernalexp.config.tooltip.dwarfAttackPlayer": "黑石侏儒将攻击玩家",
+  "infernalexp.config.tooltip.blindsightAttackGlowsquito": "黑石盲蛙将攻击荧光蚊",
+  "infernalexp.config.tooltip.blindsightAttackPlayer": "黑石盲蛙将攻击玩家",
+  "infernalexp.config.tooltip.giantAttackMagmaCube": "玄武岩巨人将攻击岩浆怪",
+  "infernalexp.config.tooltip.embodyAttackPiglin": "嗜骨沙鬼将攻击猪灵",
+  "infernalexp.config.tooltip.embodyAttackPlayer": "嗜骨沙鬼将攻击玩家",
+  "infernalexp.config.tooltip.volineAttackFireResistance": "熔岩吞噬兽将攻击抗火实体",
+  "infernalexp.config.tooltip.volineAttackPlayer": "熔岩吞噬兽将攻击玩家",
+  "infernalexp.config.tooltip.volineAttackMagmaCube": "熔岩吞噬兽将攻击小型岩浆怪",
+  "infernalexp.config.tooltip.useHogchops": "疣猪兽掉落的猪肉将被疣猪肉替换",
+  "infernalexp.config.tooltip.glowsilkSpeed": "设置荧丝蛾的飞行速度",
+  "infernalexp.config.tooltip.useThrowableBricks": "是否可投掷红砖",
+  "infernalexp.config.option.volineWastes.enable": "熔岩吞噬兽位于下界荒地",
+  "infernalexp.config.option.shroomloinCrimson.enable": "绯红菌灵位于绯红群系",
+  "infernalexp.config.option.volineCrimson.enable": "熔岩吞噬兽位于绯红群系",
+  "infernalexp.config.option.warpbeetleWarped.enable": "诡异甲虫位于诡异群系",
+  "infernalexp.config.option.giantDeltas.enable": "玄武岩巨人位于玄武岩三角洲",
+  "infernalexp.config.option.embodySSV.enable": "嗜骨沙鬼位于灵魂沙峡谷",
+  "infernalexp.config.option.glowsilkGSC.enable": "荧光蛾位于荧石峡谷",
+  "infernalexp.config.option.glowsilkDeltas.enable": "荧光蛾位于三角洲",
+  "infernalexp.config.option.glowsilkCrimson.enable": "荧光蛾位于绯红群系",
+  "infernalexp.config.tooltip.volineWastes.enable": "熔岩吞噬兽将生成在下界荒地",
+  "infernalexp.config.tooltip.shroomloinCrimson.enable": "绯红菌灵将生成在绯红森林",
+  "infernalexp.config.tooltip.volineCrimson.enable": "熔岩吞噬兽将生成在绯红森林",
+  "infernalexp.config.tooltip.warpbeetleWarped.enable": "诡异甲虫将生成在诡异森林",
+  "infernalexp.config.tooltip.giantDeltas.enable": "玄武岩巨人将生成在玄武岩三角洲",
+  "infernalexp.config.tooltip.embodySSV.enable": "嗜骨沙鬼将生成在灵魂沙峡谷",
+  "infernalexp.config.tooltip.glowsilkGSC.enable": "荧光蛾将生成在荧石峡谷",
+  "infernalexp.config.tooltip.glowsilkDeltas.enable": "荧光蛾将生成在玄武岩三角洲",
+  "infernalexp.config.tooltip.glowsilkCrimson.enable": "荧光蛾将生成在绯红森林",
+  "infernalexp.config.option.biomesListIsWhitelist": "白名单或黑名单",
+  "infernalexp.config.option.biomesList": "群系列表",
```

```suggestion
  "infernalexp.config.option.biomesList": "生物群系列表",
```
无特殊情况，biome 应当为生物群系，自行检查其他地方。